sdks/python/apache_beam/runners/internal/names.py

```diff
@@ -20,8 +20,11 @@
 # All constants are for internal use only; no backwards-compatibility
 # guarantees.
 
+DATAFLOW_SDK_TARBALL_FILE = 'dataflow_python_sdk.tar'
```

Comment:
Let's rename this to:
 
`STAGED_SDK_SOURCES_FILENAME = 'dataflow_python_sdk.tar'  # Current value is hardcoded in Dataflow internal infrastructure; please don't change without a review from Dataflow maintainers.`